Prepare release 1.0.0-rc.0 (#65)
GPP-Woo/GPP-zoeken: CHANGELOG.rst

Release notes
=============

1.0.0-rc.0 (2025-03-12)
=======================

> We proudly announce the first release candidate of GPP-Zoeken!

Reviewer: `GPP-zoeken` is what it should be. You can update that everywhere indeed.

> We proudly announce the first release candidate of GPP-Zoeken!

Author: Done!

The 1.0 version of this component is ready for production. It provides the minimal
functionalities to be able to comply with the WOO legislation in your organization.

Features
--------

* Admin panel for technical and functional administrators
    - Manage API clients and user accounts.
    - Configure connections to external services, like a Documents API and OpenID
      Connect provider.
* JSON API for manipulating indexing, deleting and searching publications and documents

>   to/from Elasticsearch. This includes the function to index and search the contents
>   of a file.

Reviewer: ```suggestion
  to/from Elasticsearch, with the ability to search through the text contents of files.
```
